README.md

```diff
@@ -43,7 +49,7 @@ AutoGVP Requirements (recommended to place all in the `data/` folder):
 
 ### Custom workflow example run
 1. [Prepare input files](https://github.com/diskin-lab-chop/AutoGVP/wiki/User-Guide#custom-input-workflow---step-by-step) by running VEP, ANNOVAR, InterVar, and AutoPVS1.
-2. Run `select-clinVar-submissions.R` with desired conflict resolution parameter ("latest", "most_severe") and optional ClinGen Concept ID list.
+2. Download [database files](https://github.com/diskin-lab-chop/AutoGVP/blob/main/scripts/download_db_files.sh) and run `select-clinVar-submissions.R` with desired conflict resolution parameter ("latest", "most_severe") and optional ClinGen Concept ID list.
```

Comment:
isn't conflict resoluation parameter only required when ClinGen ConceptID list is provided?